client/src/partials/fiscal/fiscal.update.js

```diff
@@ -0,0 +1,79 @@
+angular.module('bhima.controllers')
+.controller('FiscalUpdateController', FiscalUpdateController);
+FiscalUpdateController.$inject = [
+    '$state', 'ScrollService', 'FiscalService', 'NotifyService', 'util'
+];
+
+/**
+ * This controller is responsible for creating a debtor group. It provides
+ * utility functions for submission and error handling.
+ *
+ * @todo refactor code to remove redundant features introduced previously
+ *
+ * @module fiscal/update
+ */
+function FiscalUpdateController($state, ScrollTo, fiscalService, Notify, util) {
```

Comment:
Please make sure that you have used all the services you imported.  If you did not use them, remove them!